src/modules/job-manager/jobtap.c

```diff
@@ -221,6 +221,38 @@ int jobtap_get_priority (struct jobtap *jobtap,
     return rc;
 }
 
+int jobtap_validate (struct jobtap *jobtap,
+                     struct job *job,
+                     char **errp)
+{
+    int rc;
+    flux_plugin_arg_t *args;
+    const char *errmsg = NULL;
+
+    if (!jobtap->plugin)
+        return 0;
+    if (!(args = jobtap_args_create (jobtap, job)))
+        return -1;
```

Comment:
Since the default plugin doesn't offer `job.validate`, should there maybe be test for  `flux_plugin_get_handler() == NULL` and an early return here to avoid a bit of extra work in `jobtap_args_create()`?

Reply:
Good point in general! This will be more tricky when we allow a list of plugins, but certainly helps now I think so I've added it.